doc/tutorials/tutorial_X8.rst

```diff
@@ -496,3 +473,43 @@ compilation:
 ...     ops.MeasureFock() | q
 CircuitError: Incorrect squeezing value(s) (r, phi)={(1.336, 0.0), (0.177, 0.0), (0.818, 0.0)}.
 Allowed squeezing value(s) are (r, phi)={(1.0, 0.0), (0.0, 0.0)}.
+
+
+Appendix
+--------
+
+.. note::
+
+    **What's in a name?**
+
+    ``X8``, rather than being a single hardware chip, corresponds to a family
+    of physical devices, with individual names including ``X8_01``.
+
+    The name of each photonic chip, ``X8_01``, provides several clues to the underlying
+    chip structure and architecture.
+
+    At Xanadu, photonic devices are named using the nomenclature ``AM_YY``, where:
+
+    * ``M`` is the **mode number**, the number of modes available on the chip. ``X8_01``
+      has 8 available modes.
+
+    * ``A`` is the **architecture number.** This chip belongs to the ``X``-series of photonic
+      devices. Devices in this series are characterized by the following properties:
+
+      - The available modes :math:`M` are divided into two main groups, signal modes
+        (numbered :math:`[0, M/2-1]`) and idler modes (numbered :math:`[M/2, M-1]`).
+
+      - Initial two-mode squeezed states entangle the signal modes and the idler modes
+        in a 'cascade'; entangled mode pairs are :math:`[(0, M/2), (1, M/2+1), \dots, (M/2-1, M-1)]`.
+
+      - An arbitrary :math:`M/2\times M/2` unitary can then be applied to both the signal and idler modes.
+
+      - Finally, the modes terminate with photon-number resolving detectors.
+
+    * ``YY`` is the **chip ID**. It specifies a particular chip instance chosen for executing the jobs.
+      While all chips in the same series have the same architecture, specific chips may be characterized
+      by hardware imperfections, including loss and gate fidelity.
+
+    While ``X8`` is used here to specify *any* chip in the ``X8``-series (including ``X8_01``),
+    you may also specify a specific chip ID (such as ``X8_01``) during program compilation
+    and execution.
```

Comment:
Is it worth thinking about not having this part at all?

> ``X8`` is an 8-mode chip, with the following restrictions:

This sentence from further up above, the following paragraph, and image I think are enough for the user to know what's going on at this point.

I know this part might have taken some time to write up, but maybe it can be reused in a future iteration.

Reply:
I agree, it isn't necessary. It would be nice to find a home for this note box though, as it is useful info for research (i.e, you want to guarantee you are submitting to the same chip in case defaults change, for reproducibility).

Reply:
Yep, please remove for now from this page

Reply:
Note also that there's a difference in explaining the physical chip architecture and explaining the naming scheme. First one is fine to have on this page, but second one not necessary